object-group-game/GetConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace object_group_game
+{
+    public static class GetConfiguration
+    {
+        public static string GetSQL()
```

Comment:
I guess this method could be useful, but it should work with already-read data, not to read it. You could either drop this method or change it so it would return SQL Settings string from the Settings dictionary